beetsplug/fetchart.py

```diff
@@ -61,29 +55,25 @@ def __init__(self, log, path=None, url=None, source='',
         self.path = path
         self.url = url
         self.source = source
-        self.check = None
+        self.downscale = False
+        self.downsize = False
+        self.reformat = False
         self.match = match
         self.size = size
 
     def _validate(self, plugin):
         """Determine whether the candidate artwork is valid based on
         its dimensions (width and ratio).
```

Comment:
It looks like this method now has side effects. That is, it assigns to the `self.{downscale,downsize,reformat}` fields that otherwise default to false. This should probably be documented here so it's clear that there is a stateful update happening.